### PR TITLE
DS-12828: Adds prereq with link to KB solution, clarifies why to clon…

### PR DESCRIPTION
**Pull request by bredamc**
…e repo

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/configuring-distributed-workloads.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@ To configure the distributed workloads feature for your data scientists to use i
 * You have logged in to {openshift-platform} with the `cluster-admin` role.
 * You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
 * You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/index.html[Ray documentation].
+* You have removed all external CodeFlare Operators, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift Data Science Cluster].
```
Comment (jbyrne-redhat) [inline modules/configuring-distributed-workloads.adoc:13]: I wonder if _external_ is the most readily understood term here. It makes me ask "external to what?", even if we do link to a KB article for more information.

As an alternative, perhaps "You have removed _any previously installed instances_ of the CodeFlare Operator..."?

**Comment by sutaakar** [comment]
Looks fine to me

**Comment by bredamc** [comment]
Thanks @jbyrne-redhat! I've made the update you suggested, can you please approve if you're happy?

**Review by jbyrne-redhat — APPROVED** [review]
LGTM, @bredamc.

**Pull request merged**